cg/cli/workflow/mip_dna/base.py

```diff
@@ -101,15 +102,9 @@ def mip_dna(
         )
         # Commit the updates to request flowcells
         dna_api.db.commit()
-        return
-
-    # Invoke full workflow
-    context.invoke(link, case_id=case_id)
-    context.invoke(panel, case_id=case_id, dry_run=dry_run)
-    context.invoke(config_case, case_id=case_id, panel_bed=panel_bed, dry_run=dry_run)
-    context.invoke(
-        run, case_id=case_id, priority=priority, email=email, start_with=start_with, dry_run=dry_run
-    )
+        raise FlowcellsNeededError(
```

Comment:
Should this be a log message instead? And clarify that the flowcells are being fetched and that no manual action is needed?

Reply:
This command, when ran by itself, raises error with this error message when only flowcells not on disk and are being requested. 
When this code runs as part of `start` command, it breaks the chain of commands included in start, and delivers this message as LOG.error instead. 
Finally, when invoked and raised in with `start-available`, it is excepted and error log emssage is prnted, but the crontab doesnt exit with exit code 1, and do the cron mail will not be sent (as this is a routine error and not one to be informed about)